inclusion_connect/users/admin.py

```diff
@@ -130,13 +103,54 @@ class UserChangeForm(auth_forms.UserChangeForm):
         required=False,
         widget=TemporaryPasswordWidget,
     )
+    confirm_email = forms.BooleanField(
+        label=mark_safe('<img src="/static/admin/img/icon-alert.svg" alt="True"> Confirmer l’e-mail'),
+        disabled=True,
+        required=False,
+        widget=forms.HiddenInput(),
+    )
+
+    def __init__(self, *args, instance, **kwargs):
+        super().__init__(*args, instance=instance, **kwargs)
+        email = self.fields["email"]
+        email.label = "Adresse e-mail vérifiée"
+
+        email_to_validate = instance.email_to_validate
+        if email_to_validate:
+            confirm_email = self.fields["confirm_email"]
+            unverified_email = email_to_validate[0].email
+            confirm_email.disabled = False
+            confirm_email.widget = ConfirmEmailWidget(unverified_email=unverified_email)
+
+    def clean_email(self):
+        if self.initial["email"] != self.cleaned_data.get("email") and self.cleaned_data.get("email") == "":
+            raise ValidationError("Vous ne pouvez pas supprimer l'adresse e-mail de l'utilsateur.")
+        return self.cleaned_data.get("email")
```

Comment:
Évite deux dict lookup :)
```suggestion
        new_email = self.cleaned_data.get("email")
        if self.initial["email"] != new_email and new_email == "":
            raise ValidationError("Vous ne pouvez pas supprimer l'adresse e-mail de l'utilsateur.")
        return new_email
```